src/containers/SinglePost.js

```diff
@@ -141,7 +142,7 @@ const setupPage = (store, id, query, action) => {
   }
 
   return Promise.all([
-    saveCurrentCommunityId(dispatch, communityId, userId),
+    saveCurrentCommunity(dispatch, community, userId),
```

Comment:
trying to understand the change from `saveCurrentCommunityId` to `saveCurrentCommunity` -- is it necessary? this appears to be the only case where passing community rather than id would make a difference, but in this case (where no community is found for some reason) we'd want to display the default favicon, no?